docs/Reference/ID-Prefix-Reservation.md

```diff
@@ -0,0 +1,154 @@
+
+### Prefix resveration visual indicator
+When a package comes from a reserved prefix, you will see the below visual indicators on the [NuGet.org](https://www.nuget.org/) gallery and in Visual Studio 2017 version 15.4 or later:
+
+**NuGet.org Gallery**
+![NuGet.org Gallery](media/nuget-gallery-reserved-prefix.png)
+
+**Visual Studio**
+![Visual Studio](media/visual-studio-reserved-prefix.png)
+
+## ID prefix reservation application process
+To apply for a prefix reservation, follow the below steps. 
+1. Review the acceptance [criteria for prefix ID reservation](#id-prefix-reservation-criteria).
+2. Determine the namespaces you want to reserve, in addition to any [advanced prefix reservation scenarios](#advanced-prefix-reservation-scenarios) you may require.
+3. Send a mail to [account@nuget.org](mailto:account@nuget.org) with the owner display name on [NuGet.org](https://www.nuget.org/), as well as any reserved prefixes you are requesting. If you are delegating prefix subsets to multiple owners, make sure you mention all owner display names and prefix subsets.
+
+After the application is submitted, you will be notified of acceptance or rejection (with the criteria that caused rejection). We may need to ask additional identifying questions to confirm owner identity. 
+
+### ID prefix reservation criteria
+When reviewing any application for ID prefix reservation, the [NuGet.org](https://www.nuget.org/) team will evalaute the application against the below criteria. Not all criteria needs to be met for a prefix to be reserved, but the application may be denied if there is not substantial evidence of the criteria being met (with an explanation given):
+1. Does the package ID prefix properly and clearly identify the package owner?
+2. Are a significant number of the packages that have already been submitted by the owner under the package ID prefix?
+3. Is the package ID prefix something common that should not belong to any individual owner or organization?
+4. Would *not* reserving the package ID prefix cause ambiguity and confusion for the community?
+5. Are the identifying properties of the packages that match the package ID prefix clear and consistent (especially the package author)?
+
+## 3rd party feed provider scenarios
+If a 3rd party feed provider is interested in implementing their own service to provide prefix reservations, you can do so by modifying the search service in the NuGet v2 or v3 feed providers. The addition in the feed search service is to add the *verified* property, with examples for both the v2 and v3 feeds below. 
```

Comment:
I checked with Patricio and the client supports the prefix reservation property on the v3 search feed only. You can get rid of the v2 feed changes from here and below.

Reply:
Updated